lib/ansible/plugins/action/copy.py

```diff
@@ -25,7 +25,7 @@
 import tempfile
```

Comment:
so for this file i'm proposing this instead https://gist.github.com/bcoca/d536584c5b74bde069dc709375150364